packages/framework/src/Model/Product/Search/AggregationResultToProductFilterCountDataTransformer.php

```diff
@@ -26,7 +27,7 @@ public function translateAbsoluteNumbers(array $aggregationResult): ProductFilte
      * @param array $aggregationResult
      * @return int[]
      */
-    protected function getFlagCount(array $aggregationResult): array
+    public function getFlagCount(array $aggregationResult): array
```

Comment:
this is also a bc-break 
https://symfony.com/doc/current/contributing/code/bc.html#changing-classes

Reply:
Removed, thanks